.github/workflows/integration.yml

```diff
@@ -21,69 +21,65 @@ env:
 
 jobs:
   package-build:
```

Comment:
We should probably change the name of this now. The name shows up as `Integration tests / package-build`, which made me go looking for a separate "run" or "execute" type job.